src/popup/components/Modals/SpendSuccess.vue

```diff
@@ -78,6 +78,9 @@ export default {
     ...mapState('fungibleTokens', ['availableTokens']),
     ...mapGetters(['getTxAmountTotal', 'getTxSymbol', 'getExplorerPath']),
     ...mapGetters('names', ['getPreferred']),
+    isAE() {
+      return this.getSymbol() === 'AE';
```

Comment:
I suggest to create constant for the `'AE'` value in the constants.js file and use this constant here.